modules/core/src/main/scala/fs2/kafka/KafkaProducer.scala

```diff
@@ -8,7 +8,7 @@ package fs2.kafka
 
 import cats.Apply
 import cats.effect._
-import cats.effect.concurrent.Deferred
+import cats.effect.kernel.Deferred
```

Comment:
import from `core`, `kernel` imports ought to only be used when `core` is not a dependency

Reply:
do we actually need `core` as a dependency?

Reply:
you already have it from fs2